fvm/src/gas/price_list.rs

```diff
@@ -1047,22 +1076,30 @@ impl Rules for WasmGasPrices {
             | Operator::Return
             | Operator::Else
             | Operator::End => Ok(InstructionCost::Fixed(0)),
-            Operator::MemoryGrow { .. } => Ok({
-                // Saturating. If there's an overflow, we'll catch it later.
-                let gas_per_page =
-                    self.memory_expansion_per_byte_cost * wasmtime_environ::WASM_PAGE_SIZE;
-
-                let expansion_cost: u32 = gas_per_page
-                    .as_milligas()
-                    .try_into()
-                    .context("memory expansion cost exceeds u32")?;
-                match expansion_cost
-                    .try_into().ok() // zero or not zero.
-                {
-                    Some(cost) => InstructionCost::Linear(0, cost),
-                    None => InstructionCost::Fixed(0),
-                }
-            }),
+            // Memory related instructions...
+            Operator::TableInit { .. } | Operator::TableCopy { .. } => linear_cost(
+                Gas::zero(),
```

Comment:
No base?

Reply:
Hm. Yeah, I'll make it configurable. These are all zero right now anyways.

Reply:
Done.